.travis.yml

```diff
@@ -5,36 +5,39 @@ php:
   - 5.4
   - 5.5
   - 5.6
+  - 7.0
   - hhvm
 
+sudo: false
+
+cache:
+  directories:
+    - $HOME/.composer/cache/files
+
 env:
-  - SYMFONY_VERSION=2.5.*
+  - SYMFONY_VERSION=2.7.*
 
 matrix:
   allow_failures:
-    - env: SYMFONY_VERSION=dev-master
-    - php: 5.6
     - php: hhvm
   include:
-    - php: 5.5
-      env: SYMFONY_VERSION=2.0.*
-    - php: 5.5
-      env: SYMFONY_VERSION=2.1.*
-    - php: 5.5
-      env: SYMFONY_VERSION=2.2.*
-    - php: 5.5
+    - php: 5.6
       env: SYMFONY_VERSION=2.3.*
-    - php: 5.5
-      env: SYMFONY_VERSION=2.4.*
-    - php: 5.5
-      env: SYMFONY_VERSION=dev-master
+    - php: 5.6
+      env: SYMFONY_VERSION=2.8.*@dev
+    - php: 5.6
+      env: SYMFONY_VERSION=3.0.*@dev
+    - php: 5.3
+      env: COMPOSER_FLAGS="--prefer-lowest"
 
-before_script:
+before_install:
+  - sh -c 'if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi;'
   - git config --global user.email "test@test.com"
   - git config --global user.name "John Doe"
   - composer self-update
-  - composer require symfony/console:${SYMFONY_VERSION} symfony/yaml:${SYMFONY_VERSION} symfony/process:${SYMFONY_VERSION} --no-update
-  - composer update symfony/console symfony/yaml symfony/process
+  - if [ "$SYMFONY_VERSION" != "" ]; then composer require "symfony/symfony:${SYMFONY_VERSION}" --no-update; fi;
```

Comment:
Yeah, I've used your setup as a start (as it was also what i needed) but ran into that problem. So maybe you know a fix for that or otherwise it's no solution to switch here to specific components and we just need to keep the symfony/symfony build requirement